wrapper/src/main/java/software/amazon/jdbc/AwsWrapperProperty.java

```diff
@@ -33,6 +33,7 @@ public AwsWrapperProperty(
     this(name, defaultValue, description, required, (String[]) null);
   }
 
+  @SuppressWarnings({"nullness:argument","nullness:assignment"})
   public AwsWrapperProperty(
       String name,
```

Comment:
Probably it makes sense to make it @NonNull

Reply:
There are a number of places where the default value is null. 
We would have to change the code to be compliant.